recipes/python-ninja/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "python-ninja" %}
+{% set version = "1.10.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/n/ninja/ninja-{{ version }}.tar.gz
+  sha256: e1b86ad50d4e681a7dbdff05fc23bb52cb773edb90bc428efba33fa027738408
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - scikit-build
+  run:
+    - python
+    - scikit-build
+
+test:
+  imports:
+    - ninja
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://ninja-build.org/
+  summary: Ninja is a small build system with a focus on speed
+  license: Apache-2.0
+  license_file: LICENSE_Apache_20
+
+extra:
+  recipe-maintainers:
+    - weiji14
```

Comment:
Would anyone else like to be a maintainer for this `python-ninja` recipe? Cc @Anthchirp